src/components/v2/MyAccount/MyAccountStyles.ts

```diff
@@ -0,0 +1,105 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useMyAccountStyles = () => {
+  const theme = useTheme();
+  return {
+    root: css`
+      font-size: 20px;
```

Comment:
Since only the title uses a font size of 20px, I'd suggest not applying it to the whole tag but instead wrap the title inside a `Typography` component and apply the corresponding variant to it (`h4` I believe).